examples/caching/weather.js

```diff
@@ -0,0 +1,6 @@
+const URL = 'https://goweather.herokuapp.com/weather/'
```

Comment:
Can you please move this to the file with the rest of the code plz? It's ~25 LOC anyway